accepted/PSR-12-extended-coding-style-guide.md

```diff
@@ -461,7 +466,7 @@ In the argument list, there MUST NOT be a space before each comma, and there
 MUST be one space after each comma.
 
 Method and function arguments with default values MUST go at the end of the argument
-list.
+list, before the variable length argument lists.
```

Comment:
I don't know if this can be considered a new rule or a simple clarification, depending on the behavior of the PHP engine. Ping @dragonmantank.

Reply:
So far, they need to be placed in the end (which is the only logical place they can be set to ^^).

https://www.php.net/manual/en/functions.arguments.php#functions.variable-arg-list